flow/src/main/scala/org/alephium/flow/validation/TxValidation.scala

```diff
@@ -245,8 +245,8 @@ object TxValidation {
       }
     }
 
-    private def checkGasPrice(gas: U256): Boolean = {
-      gas > U256.Zero && gas < ALF.MaxALFValue
+    private def checkGasPrice(gasPrice: GasPrice): Boolean = {
+      gasPrice.value > U256.Zero && gasPrice.value < ALF.MaxALFValue
```

Comment:
just some random thought, we could have an extention of companion object in order to have:
```
 gasPrice > GasPrice.Zero
```

Reply:
GasPrice extends `Ordered`, but `GasPrice.Zero` should not be supported, as it's invalid. I feel the current one is better